File: src/main/java/sen/trainee/entity/Trainee.java
package sen.trainee.entity;

import java.time.LocalDate;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@ToString
@Builder
@Entity
public class Trainee {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	private String name;
	private String emailId;
	private long contactNo;
	private LocalDate dateOfBirth;
	private String city;
	private String password;
	private String aboutUs;
	

}

File: src/app/trainee.ts
export class Trainee {
    id: number;
    name: string;
    emailId: string;
    contactNo: number;
    dateOfBirth: Date;
    city: string;
    aboutUs: string;
    password: string;

    constructor() {
        this.id = 0;
        this.name = '';
        this.dateOfBirth = new Date();
        this.emailId = '';
        this.contactNo = 0;
        this.aboutUs = '';
        this.password = '';
    }

}
